docs/getstarted/getting-started.md

```diff
@@ -0,0 +1,277 @@
+
+    ![Screenshot that shows the integrated terminal in the Panel area.](images/getting-started/vscode-terminal.png)
+
+    You can choose between different shells, such as PowerShell, Command Prompt, or Bash, depending on your operating system configuration.
+
+### Access commands with the Command Palette
+
+Many of the commands in VS Code are available through the Command Palette. For example, enter *Create new file* in the Command Palette to create a file in your workspace, or enter *Git* to view the list of Git actions you can trigger.
+
+1. Select **View** > **Command Palette** or press `kb(workbench.action.showCommands)` to open the **Command Palette**.
+
+    ![Screenshot that shows the Command Palette, listing the entries for 'Create new file'.](images/getting-started/command-palette.png)
+
+    > **Tip**: Notice that the Command Palette shows the default keyboard shortcut for commands that have one. You can use the keyboard shortcut to run the command directly.
+
+1. Press `kb(workbench.action.quickOpen)` to use the Command Palette to navigate to a file in your workspace. Start typing to filter the list.
+
+    The **Quick Open** feature enables you to quickly open a file in your workspace, or search for symbols in files. Get more info the [Command Palette](/docs/getstarted/userinterface.md#command-palette) in the VS Code documentation.
+
+    ![Screenshot that shows the Quick Open feature in the Command Palette.](images/getting-started/quick-open.png)
+
+### View status information with the Status Bar
+
+The **Status Bar** is located at the bottom of the window and shows information about the file you are editing, and the workspace you have open. For example, if the folder you opened is a Git repository, the Status Bar shows the Git status and current branch.
+
+1. Open a file by selecting it in the Explorer view.
+
+    The Status Bar shows the language mode, indentation, and the line ending of the current editor.
+
+    ![Screenshot that shows the Status Bar at the bottom of the window.](images/getting-started/status-bar.png)
+
+1. Select the indentation field in the Status Bar to modify the indentation of the current file. For example, to use tabs instead of spaces.
+
+    For a Git repository, you can select the branch name in the Status Bar and create a new branch.
+
+## Step 3: Customize the user interface
+
+Each developer has their own preferences for how their coding environment should look like. VS Code enables you to change the layout, colors, keyboard shortcuts, and nearly every other aspect of the editor through various settings.
+
+Let's start by using **Color Themes** to customize the colors in VS Code. A Color Theme affects both the VS Code user interface elements and the editor highlighting colors.
+
+To select a different Color Theme:
+
+1. Select the **Manage** button (gear icon) in the Activity Bar, and then select **Themes** > **Color Theme** to display the Color Theme picker.
```

Comment:
Screenshot might be helpful (I think this is our first time mentioning the Gear in this doc?)

Reply:
It's is. Don't want to overload with too many screenshots. Maybe I can highlight it in the Activity Bar section? Good call-out.